pages/[user]/[type].tsx

```diff
@@ -175,6 +175,20 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
     periodEndDate: eventType.periodEndDate?.toString() ?? null,
   });
 
+  const currentBookings = await prisma.booking.findMany({
+    where: {
```

Comment:
Probably should optimize this to only get times from today --> on, right? Once users have used the platform for a long time, they could have hundreds of events in the past that would probably be best to exclude.